src/components/wallet-card.js

```diff
@@ -33,6 +35,9 @@ class Wallet extends Component {
   }
 
   componentDidMount() {
+    this.props.getEthBalance()
+    this.props.getOgnBalance()
+
     this.convertEthToUsd()
```

Comment:
You may have to `await` the first two async functions because I think the `convertEthToUsd` depends on the ETH balance already being set, correct? Thinking about a race condition where sometimes it tries to convert ETH to USD before the ETH balance has been set... ?